chain/beacon/drand/drand.go

```diff
@@ -68,6 +69,10 @@ func (l *logger) Named(s string) dlog.Logger {
 	return &logger{l.SugaredLogger.Named(s)}
 }
 
+func (l *logger) AddCallerSkip(skip int) dlog.Logger {
```

Comment:
why?

Reply:
I think drand requires it as part of their logger interface.

Reply:
```
	return &logger{l.SugaredLogger.With(zap.AddCallerSkip(skip))}
```

I guess my point was, why are we stubbing this.

Reply:
Sorry, I think this was a WIP thing from the Drand folks.